iso/preseeds/preseed.cfg

```diff
@@ -24,7 +25,7 @@ d-i mirror/country string manual
 d-i mirror/http/hostname string deb.debian.org
 d-i mirror/http/directory string /debian
 d-i mirror/http/proxy string
-d-i apt-setup/local0/repository string http://deb.debian.org/debian/ bullseye main contrib non-free
+d-i apt-setup/local0/repository string http://deb.debian.org/debian/ bookworm main contrib non-free
```

Comment:
Something to be aware of regarding non-free: https://www.debian.org/releases/bookworm/amd64/release-notes/ch-information.en.html#non-free-split

We need to figure out which exactly repo do we need. It should be only firmware, but I am not completly sure.

Reply:
I think they changed the `non-free` firmware to a new module called `non-free-firmware` 
Explanation of the change of  the `non-free` to `non-free-firmware` : https://unix.stackexchange.com/questions/736065/how-do-i-install-non-free-firmware-in-debian-12-bookworm
official example: https://www.debian.org/releases/bookworm/example-preseed.txt
someone's filled out example with apt packages listed etc: https://preseed.debian.net/debian-preseed/bookworm/amd64-main-full.txt

Reply:
As far as I understood it, it wasn't renamed but what was before `non-free` now has been split in `non-free` and `non-free-firmware`.

In our case, I think we just use firmware stuff.

Reply:
ah ok i read it a different way but i think you're correct, gonna try some new settings and try a reinstall on my test node see if it works.  if you have any specific suggestions for changes to the preseed configs please go ahead and make/review said changes and i can test it too, this stuff i literally tried to learn in like an hour a couple days ago when this was released, so commits are appreciated, @pablomendezroyo approved this PR but it's definitely not ready to merge just yet since we haven't resolved the preseed issues yet. im gonna get to testing some stuff now, but ill likely need help making sure any changes arent gonna break something, but right now im just trying to make sure the package manager works, which hopefully should just need changes to make it not look for CD's i think that issue is because the new bookworm ISOs are net installers but not called or named netinst anymore so the entry `d-i apt-setup/disable-cdrom-entries boolean true` needs to be there as @3alpha mentioned on discord.  Im just not sure if other things need to be added or removed, but i guess well see, cant do much without trying lol.

Reply:
so @3alpha i. messed around a bit and got stuck in the installer during select and install software section, couldnt get much from logs, it didnt fail during the preseed either so idk what was up. what do i need to add to keep apt from looking for cd sources and for this line should i remove `non-free` part in this line and the corresponding other preseed and add `non-free-firmware` after  `contrib` instead?

Reply:
```suggestion
d-i apt-setup/local0/repository string http://deb.debian.org/debian/ bookworm main contrib non-free-firmware
```

Reply:
@3alpha let me know if this looks good, it works so far for me, no issues yet.

Reply:
@3alpha I think I resolved this conversation before committing the fix in one of my above comments that is used identically  in the `preseed_unattended.cfg` so I assume it supposed to be the same just seem to have closed it as resolved first, committing it now just double check and make sure I'm not missing something other than forgetting to commit it yesterday and instead marking this as closed.  i was checking the diff and thats how I noticed my mistake, just let me know that it was actually my mistake not me ignoring a review, webUI of GH is acting up for me today and I just want you to be sure sure before signing off on this PR.
Totally separate question, what's your hypervisor of choice?, and so you know if any work better than others specifically with Dappnode? im trying to make a final decision before putting my new hardware to work.